addon.py

```diff
@@ -38,7 +38,8 @@
 # protocol : HBBTV | RTMP
 # quality  : SQ (High) | EQ (Med) | HQ (Low)
 video_json = base_url + '/papi/tvguide/videos/stream/player/{lang}/{id}_PLUS7-{lang}/{protocol}/ALL.json'
-
+video_json = 'https://api.arte.tv/api/player/v1/config/{lang}/{id}'
+daily_json = 'https://www.arte.tv/hbbtvv2/services/web/index.php/OPA/v3/programs/{date}/de'
```

Comment:
Isn't the "de" for german?
Please bear with us, frenchies :)

Reply:
True! Now listing will be displayed in fr if language is set to FR.
Stream was already in fr see versionShortLibelle | "FR".